internal/cmd/cmd_test.go

```diff
@@ -197,3 +183,33 @@ XlW7KilKI5YkcszGoPB4RePiHsH+7trf7l8IQq5r5kRq7SKsZ41BI6s1E1PQVW93
 		assert.ErrorContains(t, err, "context not found")
 	})
 }
+
+// newTestClientConfig returns a ClientConfig that can be used to test CLI
+// commands. Most CLI commands require a login first, which saves a ClientConfig
+// to a file.
+// newTestClientConfig provides a reasonable default for most cases, removing
+// the need to perform a full login. The returned value may be modified, and then
+// should be saved to a file with writeConfig.
+// If any fields in identity are not set, they will be set to default values.
+func newTestClientConfig(srv *httptest.Server, identity api.Identity) ClientConfig {
```

Comment:
I wasn't sure about the `*httptest.Server` arg. We could pass in just a string instead, but in the future if we want to use the TLS settings from the server we might need more of it.